### PR TITLE
New dashboard widget Split Graph

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

Add documentation for new dashboard widget Split Graph
[DOCS-6057](https://datadoghq.atlassian.net/browse/DOCS-6057)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

- PM to review, do not merge.

[DOCS-6057]: https://datadoghq.atlassian.net/browse/DOCS-6057?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by aliciascott** [comment]
Thanks! Created Editorial Review for Docs team: https://datadoghq.atlassian.net/browse/DOCS-6292

File: content/en/dashboards/widgets/split_graph.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+title: Split Graph Widget
+kind: documentation
+widget_type: split_graph
+further_reading:
+- link: "/dashboards/guide/graphing_json/"
+  tag: "Guide"
+  text: "Build dashboards using JSON"
+---
+
+<div class="alert alert-info">Split Graph widgets are not supported on screenboards and <a href="https://docs.datadoghq.com/dashboards/sharing/#share-a-dashboard-by-public-url">public dashboards.</a></div>
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:11]:
```suggestion
<div class="alert alert-info">Split graph widgets are not supported on screenboards and <a href="https://docs.datadoghq.com/dashboards/sharing/#share-a-dashboard-by-public-url">public dashboards.</a></div>
```
For consistency, I've used lower case unless we're talking about the UI

File: content/en/dashboards/widgets/split_graph.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Overview
+
+Split Graph allows you to break down a query across multiple tag values to identify outliers and patterns. Use this feature to investigate metrics performance across multiple facets, compare events across multiple tags, or create dynamic visualizations.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:15]:
```suggestion
A split graph allows you to break down a query across multiple tag values to identify outliers and patterns. Use this feature to investigate metrics performance across multiple facets, compare events across multiple tags, or create dynamic visualizations. 
```

File: content/en/dashboards/widgets/split_graph.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Overview
+
+Split Graph allows you to break down a query across multiple tag values to identify outliers and patterns. Use this feature to investigate metrics performance across multiple facets, compare events across multiple tags, or create dynamic visualizations. 
+
+## Setup
+
+### Create a split graph widget
+
+Find the Split graph widget in the widget tray, under the group section, and drag it on your dashboard to create a split graph from scratch. This allows you to define both the query and the split dimensions at the same time. For more information on configuration options, see the [Configuration](#configuration) section.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:21]:
```suggestion
Find the split graph widget in the widget tray, under the group section, and drag it on your dashboard to create a split graph from scratch. This allows you to define both the query and the split dimensions at the same time. For more information on configuration options, see the [Configuration](#configuration) section.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+### Create a split graph from an existing widget
+
+You can also create a split graph by taking an existing widget and splitting it on a tag value using the "Split Graph" tab. Open the "Split Graph" tab through the following:
+- Add a new widget to your dashboard and click the "Split Graph" tab at the top of the query editor.
+- Open a widget in full screen mode by selecting either the edit or expand icons from the widget control options, then click on the "Split Graph" tab.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:27]:
```suggestion
- Open a widget in full screen mode by selecting either the edit or expand icons from the widget control options, then click on the **Split Graph** tab.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+- Open the context menu of a widget on your dashboard and select "Split Graph".
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:28]:
```suggestion
- Open the context menu of a widget on your dashboard and select **Split Graph**.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+In the "Split Graph" tab, you can configure how your graph is split, set the limit on the number of graphs, and configure the order.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:30]:
```suggestion
From the **Split Graph** tab, you can configure how your graph is split, set the limit on the number of graphs, and configure the order.  
```

File: content/en/dashboards/widgets/split_graph.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+- Open a widget in full screen mode by selecting either the edit or expand icons from the widget control options, then click on the "Split Graph" tab.
+- Open the context menu of a widget on your dashboard and select "Split Graph".
+
+In the "Split Graph" tab, you can configure how your graph is split, set the limit on the number of graphs, and configure the order.  
+1. Make configuration changes to the split by editing the split dimension, number of graphs displayed, or editing display options. Fore more information on the configuration options, see the [Configuration](#configuration) section.
+2. Click **Save to Dashboard** to create a new split graph widget at the bottom of your dashboard. Your original widget remains unchanged on your dashboard. 
+
+### Create a split graph from elsewhere in Datadog
+
+Whenever a split across multiple values is displayed in the app you can export it as a widget to a dashboard.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:36]:
```suggestion
Whenever a split across multiple values is displayed in the app, you can export it as a widget to a dashboard.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+1. Click **Export to Dashboard**.
+1. An export modal opens where you can either search for an existing dashboard to export to or create a new dashboard containing this widget.
+
+## Configuration
+
+If you create a split graph from scratch or edit an existing split graph on your dashboard, you have the option to configure the graph as well as the split. 
+
+The split graph editor is composed of two separate sections: ["Edit Graph"](#edit-graph) and ["Split Graph"](#edit-split). To add a widget title, update the text input at the top of the editor.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:44]:
```suggestion
The split graph editor is composed of two separate sections: [**Edit Graph**](#edit-graph) and [**Split Graph**](#edit-split). To add a widget title, update the text input at the top of the editor.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+**Note**: If you create a split graph from a widget, you only have the option to configure the split in the "Split Graph" tab. You can always click on the "Edit" tab to edit the query.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:46]:
```suggestion
**Note**: If you create a split graph from a widget, you only have the option to configure the split in the **Split Graph** tab. You can always click on the **Edit** tab to edit the query.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+
+{{< img src="dashboards/widgets/split_graph/split_graph_tab.png" alt="Split graph tab displays split graph configuration options" style="width:100%;" >}}
+
+### Edit graph
+
+Configure your graph query before splitting. Choose any visualization type that supports splitting and make changes to how the graphs are displayed. You can also create your query from scratch, just like in the standard query editor experience. 
+
+For more information on the individual configuration for these visualizations, see the respective documentation for supported widgets in the [Widgets][1] page.
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:54]:
```suggestion
For more information on the individual configuration for these visualizations, see the respective documentation for supported widgets on the [Widgets][1] page.
```

File: content/en/dashboards/widgets/split_graph.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+### Edit split
+
+There are several inputs that allow you to configure how to split the graph as well as split-specific display options.
+
+| Configuration input | Description    | 
+| ---  | ----------- | 
+| One graph per | This dropdown defines the dimension to split your original graph on. |
+| Limit to | Option to specify the number of graphs to display and which values should be selected. By default, the split graph widget dynamically selects values with highest average values. |
+| Sort by | Choose a metric or an attribute/facet to sort your graphs by. Selecting "custom" lets you manually select tags to display. |
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:68]:
```suggestion
| Sort by | Choose a metric or an attribute/facet to sort your graphs by. Selecting **custom** lets you manually select tags to display. |
```

File: content/en/dashboards/widgets/split_graph.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+| Show Controls | Toggle to display a sidebar with all available tag values. Manually select tag values to change the split from dynamic to static and show only the values you have selected. To go back to a dynamic behavior, clear the selection or click the **Custom** button and select "Top" or "Bottom" to re-enable sorting.|
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:69]:
```suggestion
| Show Controls | Toggle to display a sidebar with all available tag values. Manually select tag values to change the split from dynamic to static and show only the values you have selected. To go back to a dynamic behavior, clear the selection or click the **Custom** button and select **Top** or **Bottom** to re-enable sorting.|
```

File: content/en/dashboards/widgets/split_graph.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
+---
+title: Split Graph Widget
+kind: documentation
+widget_type: split_graph
```
Comment (hestonhoffman) [inline content/en/dashboards/widgets/split_graph.md:4]:
```suggestion
widget_type: split_group
```
Kinda confusing, but it looks like you have to use `split_group` for the API. This is so the tables at the bottom of the page load correctly.

**Review by hestonhoffman — APPROVED** [review]
Looks good! Made some style changes and a couple of other suggestions.

File: content/en/dashboards/widgets/split_graph.md (hunk 13 above)
Comment (estherk15) [reply content/en/dashboards/widgets/split_graph.md:4]: Thanks for catching this Heston!

**Pull request merged**